src/Controller/Organization/CommissionableAsset/AssetDeleteController.php

```diff
@@ -7,18 +7,20 @@
 use App\Entity\CommissionableAsset;
 use App\Entity\Organization;
 use App\Repository\CommissionableAssetAvailabilityRepository;
+use App\Security\Voter\CommissionableAssetVoter;
 use Doctrine\ORM\EntityManagerInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\Routing\Annotation\Route;
-use Symfony\Component\Security\Core\Exception\AccessDeniedException;
 
 /**
- * @Route("/{organization}/commissionable-assets/{asset}/delete", name="app_commissionable_asset_delete", methods={"GET"})
+ * @Route("/{asset<\d+>}/delete", name="app_organization_asset_delete", methods={"GET"})
  * @Security("asset.organization.id == organization")
+ * @IsGranted(CommissionableAssetVoter::CAN_EDIT, subject="asset")
```

Comment:
The original condition for this route was "current user is organization and is parent organization", i don't know if this was an error. Otherwise, we need another case in the voter

Reply:
Edit and Delete are the same thing here, if it was a different test before, it was an error